content/src/fieldConfig/business-formation.json

```diff
@@ -400,7 +400,7 @@
         "placeholder": "Address Line 2",
         "label": "**Address Line 2**"
       },
-      "addressCity": {
+      "foreignAddressCity": {
```

Comment:
this field shouldn't change. this is in the addressModal. this field does not represent the foreign city.